lib/Dialect/FIRRTL/LowerToRTL.cpp

```diff
@@ -97,6 +97,23 @@ static LogicalResult lower(firrtl::ConstantOp op, ArrayRef<Value> operands,
   return success();
 }
 
+static LogicalResult lower(firrtl::WireOp op, ArrayRef<Value> operands,
+                           ConversionPatternRewriter &rewriter) {
+  auto resType = op.result().getType().cast<FIRRTLType>();
+  if (auto resultType = RTLTypeConverter::convertType(resType)) {
```

Comment:
Nit: I'd recommend inverting the conditional and use an "early out" to reduce the nesting in this, e.g.:

```
auto resultType = RTLTypeConverter::convertType(resType);
if (!resultType) return failure();
```

In cases like the 'if' below, you can omit the "else" after the return to similarly reduce nesting.  That said, see the comment below